app/models/vacation.rb

```diff
@@ -60,6 +60,12 @@ class Vacation < ApplicationRecord
     where("#{approver}_approver_id": nil)
   }
 
+  scope :finished, -> {
+    where(status: :approved)
```

Comment:
what about using `approved` instead this where

Reply:
Way better, totally missed that example that used pending.